src/main/java/org/folio/bulkops/repository/RemoteFileSystemRepository.java

```diff
@@ -0,0 +1,20 @@
+package org.folio.bulkops.repository;
+
+import lombok.RequiredArgsConstructor;
+import org.folio.s3.client.FolioS3Client;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@RequiredArgsConstructor
+public class RemoteFileSystemRepository {
+
+  public final FolioS3Client folioS3Client;
+
+  public String put(String pathToNewFile, String fileNameToBeUpdated) {
```

Comment:
We are going to avoid operations with local file system, including upload, to support scaling and high availability. Method String write(String path, InputStream is); from folio-s3-client should be used for this purpose.Please, change upload to write in this implementation.